### DISCUSSION TITLE
moving from merging scATAC pipeline to Analyzing PBMC scATAC-seq pipeline

### DISCUSSION THREAD

**Discussion by pengxin2019** [stuart-lab/signac#567] [Q&A] [answered]
Hi Tim,
I have finished merging two scATAC datasets using this pipeline (https://satijalab.org/signac/articles/merging.html). Now, I am moving to the  Analyzing PBMC scATAC-seq (https://satijalab.org/signac/articles/merging.html). (By the way, feel free to correct me if you think this combination of pipeline is not proper).

However, I am stuck in the  step below (second pipeline mentioned above) since there is no peak_region_fragments column or blacklist_region_fragments column or peak_region_fragments  in the "pbmc" object.
# add blacklist ratio and fraction of reads in peaks
pbmc$pct_reads_in_peaks <- pbmc$peak_region_fragments / pbmc$passed_filters * 100
pbmc$blacklist_ratio <- pbmc$blacklist_region_fragments / pbmc$peak_region_fragments

In the merging  pipeline, the "pbmc500.counts" object does not have those three columns which is included in the  "md.500". So, I think this is a little gap between these two pipeline.  Can you give me some suggestions on how to transfer these three columns to the pbmc which is a Seurat Object? I am new to it

Thanks

**Comment by timoast** [comment, marked as answer]
Those metadata values are present in the per-cell metadata computed by 10x cellranger-atac. To include that information, set the `meta.data` field in `CreateSeuratObject()` as shown in the vignettes, or use the `AddMetaData()` function.
